src/Context/cartContext.js

```diff
@@ -0,0 +1,42 @@
+import { createContext, useReducer, useContext } from "react";
+
+const CartContext = createContext();
+
+const cartFunction = (cartState, action) => {
+    switch(action.type){
+        case "Add_to_cart":
+            return {...cartState,
+               cart: [
+                  ...cartState.cart, 
+                  {...action.payload, quantity: 1}]}
+        case "Increase_quantity":
+            return {...cartState,
+                cart: 
+                    cartState.cart.map((item) => item.id === action.payload.id ? 
+                    {...item, quantity: item.quantity + 1} : item)
+                    }
+        case "Decrease_quantity":
+            return {...cartState,
+                cart: 
+                    cartState.cart.map((item) => item.id === action.payload.id ? 
+                    {...item, quantity: item.quantity > 1 ? item.quantity - 1: item.quantity} : item)
+                    }
+                    case "Remove_from_cart":
```

Comment:
Good practice is using capital letters for cases like "REMOVE_FROM_CART".

Reply:
Okay